Mlem/App/Views/Root/Tabs/Settings/DeveloperSettingsView.swift

```diff
@@ -28,6 +34,28 @@ struct DeveloperSettingsView: View {
                 NavigationLink(String("Error Log"), destination: .settings(.errorLog))
             }
             
+            Section {
+                HStack {
+                    Text("Status")
+                    Spacer()
+                    if let backendStatus {
+                        Image(systemName: Icons.present)
+                            .foregroundStyle(backendStatus ? .themedPositive : .themedNegative)
+                    } else {
+                        ProgressView()
+                    }
+                }
+                
+                Button("Refresh") { checkBackendStatus() }
+            } header: {
+                Text("Backend")
+            } footer: {
+                if let lastBackendStatusCheck {
+                    Text("Refreshed \(lastBackendStatusCheck.formatted(date: .abbreviated, time: .standard))")
+                }
+            }
+            .task { checkBackendStatus() }
```

Comment:
```suggestion
            .onAppear { checkBackendStatus() }
```

Or, we could make `checkBackendStatus` an `async` method